src/NexusMods.App.UI/Controls/Settings/SettingEntries/SettingInteractionControls/Toggle/SettingToggleControl.axaml.cs

```diff
@@ -12,6 +12,8 @@ public SettingToggleControl()
     {
         InitializeComponent();
 
+        ClipToBounds = false;
```

Comment:
Logically I would think that this should not be needed since it should already be set by the style, but Avalonia doesn't always behave logically

Reply:
It felt a bit weird needing to put that there but this component was set to True by default 🤷

Reply:
I tested this and you can move this into the `SettingToggleControl.axaml`, but on the control definition itself, not on the ToggleSwitch. 

```xml
<reactiveUi:ReactiveUserControl
    x:TypeArguments="settingEntries:ISettingToggleViewModel" xmlns="https://github.com/avaloniaui"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:reactiveUi="http://reactiveui.net"
    xmlns:settingEntries="clr-namespace:NexusMods.App.UI.Controls.Settings.SettingEntries"
    mc:Ignorable="d"
    x:Class="NexusMods.App.UI.Controls.Settings.SettingEntries.SettingToggleControl"
    ClipToBounds="False">
```

Reply:
Oh nice, yeah that's way better.